server/files/log_file.go

```diff
@@ -0,0 +1,27 @@
+package files
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"runtime"
+	"strings"
+	"time"
+)
+
+// GetLogFile returns a file in the log directory for the current OS
+func GetLogFile(logName string) (*os.File, error) {
+	datestamp := time.Now().Format("2006-01-02_15.04.05")
```

Comment:
I've never seen a format string like that.  Usually it looks like "YYYY-MM-DD_hh.mm.ss".  Interesting.

Reply:
yeah, golang does it a bit weird. they have a 'reference date' (you can look up the docs) that you format the way you want, and it'll make any arbitrary date fit that pattern. It's kinda nice because you can do things like `.Format("In the year 2006, in the month of January")` and it will actually replace the date parts. It's actually pretty cool.